playbooks/tensorflow-volcano-kubeflow-integration-test-k8s/volcano_tfbenchmarks.yaml

```diff
@@ -0,0 +1,193 @@
+          ps_cpu: 1000m, ps_mem: 1000Mi, worker_cpu: 2000m, worker_mem: 3800Mi, minavailable: 6}
+      - { case: "case4", name: "vc-tfbenchmarks-1", ps_replicas: 2, worker_replicas: 4,
+          ps_cpu: 1000m, ps_mem: 1000Mi, worker_cpu: 2000m, worker_mem: 3800Mi, minavailable: 6}
+      - { case: "case4", name: "vc-tfbenchmarks-2", ps_replicas: 2, worker_replicas: 4,
+          ps_cpu: 1000m, ps_mem: 1000Mi, worker_cpu: 2000m, worker_mem: 3800Mi, minavailable: 6}
+      - { case: "case4", name: "vc-tfbenchmarks-3", ps_replicas: 2, worker_replicas: 4,
+          ps_cpu: 1000m, ps_mem: 1000Mi, worker_cpu: 2000m, worker_mem: 3800Mi, minavailable: 6}
+      - { case: "case4", name: "vc-tfbenchmarks-4", ps_replicas: 2, worker_replicas: 4,
+          ps_cpu: 1000m, ps_mem: 1000Mi, worker_cpu: 2000m, worker_mem: 3800Mi, minavailable: 6}
+      - { case: "case4", name: "vc-tfbenchmarks-5", ps_replicas: 2, worker_replicas: 4,
+          ps_cpu: 1000m, ps_mem: 1000Mi, worker_cpu: 2000m, worker_mem: 3800Mi, minavailable: 6}
+  tasks:
+    - name: prepare k8s namespaces for volcano and TFjobs
+      shell:
+        cmd: |
+          set -ex
+          cat <<EOF | kubectl create -f -
+          apiVersion: v1
+          kind: Namespace
+          metadata:
+           name: volcano
+           annotations:
+             scheduler.alpha.kubernetes.io/node-selector: env=system
+          spec: {}
+          status: {}
+          EOF
+
+          cat <<EOF | kubectl create -f -
+          apiVersion: v1
+          kind: Namespace
+          metadata:
+           name: vc-tfbenchmarks
+           annotations:
+             scheduler.alpha.kubernetes.io/node-selector: env=workload
+          spec: {}
+          status: {}
+          EOF
+          sed -i '/--enable-admission-plugins/ s/$/,PodNodeSelector/' /etc/kubernetes/manifests/kube-apiserver.yaml
+          sleep 30
+        executable: /bin/bash
+
+    - name: create vc-tfbenchmarks directory
+      file: path="{{ ansible_user_dir }}/workspace/vc-tfbenchmarks/{{ item }}" state=directory
+      with_items:
+        - case1
+        - case2
+        - case3
+        - case4
+
+    - name: customize volcano tfbenchmarks templates
+      become: yes
+      template:
+        dest: "{{ ansible_user_dir }}/workspace/vc-tfbenchmarks/{{ item.case }}/{{ item.name }}.yaml"
+        src: "vc-tfbenchmarks.yaml.j2"
+      vars:
+        vc_tfbenchmarks_name: "{{ item.name }}"
+        ps_replicas: "{{ item.ps_replicas }}"
+        worker_replicas: "{{ item.worker_replicas }}"
+        ps_cpu: "{{ item.ps_cpu }}"
+        ps_mem: "{{ item.ps_mem }}"
+        worker_cpu: "{{ item.worker_cpu }}"
+        worker_mem: "{{ item.worker_mem }}"
+        minavailable: "{{ item.minavailable }}"
+      with_items: "{{ cases }}"
+
+    - name: deploy volcano
+      shell: |
+        set -ex
+        mkdir -p src/volcano.sh/
+        git clone https://github.com/volcano-sh/volcano src/volcano.sh/volcano
```

Comment:
I think we should pin a specify version to instead master in order to avoid updates in master impact out demo.

Reply:
OK, will check with other colleagues.

Reply:
after confirming with mada, he suggested to use master for now